src/ceph.in

```diff
@@ -849,7 +849,11 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, ('mon', ''), ' '.join(childargs))
+
+    # implement "tell module help"
+    if len(childargs) >= 3 and childargs[0] == 'tell' and childargs[2] == 'help':
+        return do_extended_help(parser, childargs, childargs[1].split('.'), None)
```

Comment:
yes. that report an error is better. i just want to keep this PR simple before, i would like to report error. i will update my code.

```and if the component before . is not osd and the length of childargs[1].split('.') is 1, you might need to append an empty string to the end of it.```

i think it's not need to append empty string. `ceph tell mgr help` is ok.